terraform/app/modules/aks/variables.tf

```diff
@@ -131,7 +131,7 @@ locals {
 
   # AKS
   # Use the AKS ingress domain by default. Override with the DOMAIN variable is present
-  # The TEMP_DOMAIN variable takes precedence during the migration from PaaS to AKS
+  # The TEMP_DOMAIN variable takes precedence during the migration  to AKS
```

Comment:
This can be made more generic:
`# The TEMP_DOMAIN variable takes precedence, in case of a migration to a new environment for instance`